src/cart/crt_context.c

```diff
@@ -954,22 +953,21 @@ crt_context_timeout_check(struct crt_context *crt_ctx)
 					delay / 1000, delay % 1000,
 					max_delay / 1000, max_delay % 1000);
 				swim_net_glitch_update(csm->csm_ctx, self_id, delay);
```

Comment:
any issue with this function being called under csm lock? that function does some sort of its own locking as well

Reply:
Yes, many parallel xstreams increase glitch timeout. Not critical but see duplication messages in log.